ui/src/components/table/tests/TableWrapper.test.tsx

```diff
@@ -0,0 +1,116 @@
+        displayActionBtnAllRows: false,
+    };
+
+    server.use(
+        http.get('/servicesNS/nobody/-/splunk_ta_uccexample_account', () =>
+            HttpResponse.json(MockRowData)
+        )
+    );
+
+    setUnifiedConfig(SIMPLE_NAME_TABLE_MOCK_DATA);
+
+    render(
+        <Router>
+            <TableContextProvider>
+                {(<TableWrapper {...props} />) as unknown as Node}
+            </TableContextProvider>
+        </Router>
+    );
+
+    const numberOfItems = await screen.findByText('9 Items');
+    expect(numberOfItems).toBeInTheDocument();
+
+    const headerNames = ['Name', 'Actions'];
+
+    const tableHeader = document.querySelectorAll('th');
+
+    expect(tableHeader.length).toEqual(headerNames.length);
+
+    headerNames.forEach((name) => {
+        const thWithName = Array.from(tableHeader).find(
+            (thElem: HTMLElement) => thElem.dataset.testLabel === name
+        );
+        expect(thWithName).toBeTruthy();
+    });
+
+    const currentTab = SIMPLE_NAME_TABLE_MOCK_DATA.pages.configuration.tabs.find(
+        (tab) => tab.name === props.serviceName
+    );
+
+    currentTab?.entity.forEach((confEntity) =>
+        expect(screen.getByText(confEntity.label)).toBeInTheDocument()
+    );
+});
+
+it('sort items after filtering', async () => {
+    const props = {
+        page: 'configuration',
+        serviceName: 'account',
+        handleRequestModalOpen,
+        handleOpenPageStyleDialog,
+        displayActionBtnAllRows: false,
+    };
+
+    server.use(
+        http.get('/servicesNS/nobody/-/splunk_ta_uccexample_account', () =>
+            HttpResponse.json(MockRowData)
+        )
+    );
+
+    setUnifiedConfig(TABLE_CONFIG_WITH_MAPPING);
+
+    render(
+        <Router>
+            <TableContextProvider>
+                {(<TableWrapper {...props} />) as unknown as Node}
+            </TableContextProvider>
+        </Router>
+    );
+
+    const numberOfItems = await screen.findByText('Custom Text');
+    expect(numberOfItems).toBeInTheDocument();
+
+    const customHeader = document.querySelector('[data-test-label="Custom Text"]');
+    expect(customHeader).toBeInTheDocument();
+
+    const defaultOrder = document.querySelectorAll('[data-column="custom_text"]');
+    const mappedTextDefaultOrder = Array.from(defaultOrder).map((el: Node) => el.textContent);
+    expect(mappedTextDefaultOrder).toMatchSnapshot();
+
+    await userEvent.click(customHeader!);
+
+    const allCustomTextsAsc = document.querySelectorAll('[data-column="custom_text"]');
+    const mappedTextAsc = Array.from(allCustomTextsAsc).map((el: Node) => el.textContent);
+
+    expect(mappedTextAsc).toMatchSnapshot();
+
+    await userEvent.click(customHeader!);
+
+    const allCustomTextsDesc = document.querySelectorAll('[data-column="custom_text"]');
+    const mappedTextDesc = Array.from(allCustomTextsDesc).map((el: Node) => el.textContent);
+
+    expect(mappedTextDesc).toMatchSnapshot();
```

Comment:
From my point of view, [inline snapshots](https://jestjs.io/docs/snapshot-testing#inline-snapshots) are more useful here. You don't have to jump to another file to see what is there, and the size is not big